src/wolfgame.py

```diff
@@ -568,14 +568,20 @@ def replace(var, wrapper, message):
         target, _ = users.complete_match(rest[0], pl)
 
         if target is None:
-            wrapper.pm(messages["target_not_playing"].format(" longer" if target in var.DEAD else "t"))
+            wrapper.pm(messages["target_not_playing"].format("t"))
```

Comment:
Split this message up into two keys, one with "not playing" and one with "no longer playing"

The way it is right now is fugly and needs to die